bca4abm/bca4abm.py

```diff
@@ -113,6 +116,8 @@ def read_assignment_spec(fname):
 # generator for chunked iteration over dataframe by chunk_size
 def chunked_df(df, rows_per_chunk, trace_rows=None):
 
+    print(("rows_per_chunk ", rows_per_chunk))
```

Comment:
need to remove this

Reply:
at least make it a logger debug statement (activitysim does a lot of tracing of this stuff - can we combine that code?)

Reply:
oops this was leftover from debugging. I'll open an issue about combining the asim/bca chunking logic since they do share a lot.